source/Coop.sln

```diff
@@ -1,7 +1,7 @@
 ﻿

```

Comment:
Lots of scary changes in here - Are you running VS 2022? shouldn't be any reason for these project ids and such to change

Reply:
Yup, VS 2022. I will revert the changes.